src/gui/tagfiltermodel.cpp

```diff
@@ -105,8 +105,6 @@ TagFilterModel::TagFilterModel(QObject *parent)
 
 TagFilterModel::~TagFilterModel()
```

Comment:
Right, pardon me. But nevertheless why don't you save a couple of lines and declarre it as default then?